lms/static/js/student_account/views/LoginView.js

```diff
@@ -19,12 +21,15 @@
                 formType: 'login',
                 requiredStr: '',
                 submitButton: '.js-login',
+                formSuccessTpl: formSuccessTpl,
+                formStatusTpl: formStatusTpl,
```

Comment:
Consistency with the FormView. Also, so that any view that inherits from LoginView can inherit them as well.